packages/myst-transforms/src/enumerate.ts

```diff
@@ -9,7 +9,8 @@ import { createHtmlId, fileWarn, normalizeLabel, setTextAsChild, copyNode } from
 
 const TRANSFORM_NAME = 'myst-transforms:enumerate';
 
-type ResolvableCrossReference = CrossReference & {
+type ResolvableCrossReference = Omit<CrossReference, 'kind'> & {
+  kind?: TargetKind | string;
```

Comment:
@fwkoch I think we should go back and remove the "kind" from the myst-spec on CrossRefererence, right now that is too docutils specific.